src/components/atoms/ChatMessage/ChatMessage.scss

```diff
@@ -6,6 +6,7 @@ $my-text-border-radius: 28px 28px 4px 28px;
 .chat-message {
   align-self: flex-start;
   display: flex;
+  align-items: flex-start;
```

Comment:
@mike-lvov How does adding `align-items` here differ from `align-self`?

- https://www.w3schools.com/cssref/css3_pr_align-self.asp
  - > The align-self property specifies the alignment for the selected item inside the flexible container.
  - > Note: The align-self property overrides the flexible container's align-items property.
  
Is `align-self` still needed here?

Reply:
They target different items. Align-self targets the actual class. align-items targets its children

Reply:
Right, that makes sense.